src/resources/MachineLearning/ModelDetailedInfo/ModelDetailedInfo.ts

```diff
@@ -0,0 +1,11 @@
+import API from '../../../APICore.js';
+import Resource from '../../Resource.js';
+import {ModelWithDetails} from './ModelDetailedInfoInterfaces.js';
+
+export default class ModelDetailedInfo extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/modeldetailedinfo`;
+
+    get(modelId: string) {
```

Comment:
Missing test coverage for this new method

Reply:
fixed